backend/src/chat_server/utils.ts

```diff
@@ -35,25 +35,31 @@ export async function isBanned(username: string) {
   );
 }
 
-function createRegexMask(word) {
+function createRegexMask(word, stage) { //Stage = 1 Normal RegEx filter; //Stage = 2 Regex with boundary
```

Comment:
Bien `stage` nay em cung fai dung ten cu the hon, vd nhu sau
```suggestion
function createRegexMask(word, opts: { with_word_boundary }) { //Stage = 1 Normal RegEx filter; //Stage = 2 Regex with boundary
```
thay vi dung constant 1, 2.
Nen dat ten functions, variables, parameters clear & easy to other to use